### ISSUE TITLE
Create a fork of this repo for covidsimteam org called 'edcd-covhub'

### ISSUE THREAD

**Issue by risavkarna** [nepwork/admin-ui#10]
This fork will be a simpler version of the dashboard containing only the features requested by EDCD, Nepal.

Currently this includes the following sections from the sidebar:

Data Entry -> Returnees Entry Forms:
- Point of Entry
- Quarantine Facility
- RDT Lab
- PCR Lab
- Isolation
- Hospital
- Municipality

Aggregates -> Returnees Test Tables
- PCR Tests
- RDT Tests
- Returnees

Visual Stats -> COVID-19 Maps
- Kathmandu Valley (Should be changed to 'Region')
- Nepal 

The home page will be finalized soon. It may contain important maps and charts.

**Comment by risavkarna** [comment]
@Cryptokripa Do we have all forms in Nepali?

**Comment by Cryptokripa** [comment]
No, I had an emergency so couldn’t finish it. I just pushed what I could do
at that time and haven’t gotten time to work on it since.
